pcbuilder_crawlers/pipelines.py

```diff
@@ -6,19 +5,12 @@
 
 
 class Pipeline(object):
-    def __init__(self):
-        self.root_url = config.get("API_URL")
-        # self.api_url = ("{root_url}category/{category_name}/product"
-        #                 "/{product_id}" .format(root_url=self.root_url))
 
     def process_item(self, item, spider):
         template = TemplateInterface()
         category = template.translate_category(item['category'])
-        if spider is HardwareInfoSpider:
-            self.post_item(item)
-        else:
-            self.post_price(item)
-        return item
+        temp = template.get_template(category)
```

Comment:
Please do this bit in `post_item(self, item):` . This is specific to the hardware info items.